tests/integration/sqcmds/panos-samples/address.yml

```diff
@@ -1067,8 +1067,8 @@ tests:
   data-directory: tests/data/basic_dual_bgp/parquet-out
   marks: address show panos filter
   output: '[{"namespace": "dual-bgp", "hostname": "spine01", "ifname": "lo", "ipAddressList":
-    ["10.0.0.21/32"], "macaddr": "00:00:00:00:00:00", "ip6AddressList": "2001:db8::21/128",
-    "state": "up", "vrf": "default", "timestamp": 1616638471112}]'
+    ["10.0.0.21/32"], "macaddr": "00:00:00:00:00:00", "ip6AddressList": ["2001:db8::21/128",
+    "::1/128"], "state": "up", "vrf": "default", "timestamp": 1616638471112}]'
 - command: address show --address="2001:db9::21/128"  --format=json
   data-directory: tests/data/basic_dual_bgp/parquet-out
   marks: address show panos filter
```

Comment:
For everything in the dual-bgp samples dir, don't we need cumulus in the marks too?